Automattic/woocommerce-payments
# Fixes flaky UPE E2E tests

Fixes #6322 

#### Changes proposed in this Pull Request


# E2E Errors Begone

The two tests "WC - latest | upe - shopper" and "WC - latest | UPESplit - shopper" keep failing infrequently but frequently enough on GH actions for WCPay PRs. These changes make a few modifications to amend this distressing and rather annoying situation. 

Firstly SEPA enablement is removed for split-UPE E2E tests, since the split-UPE feature is not yet fully compatible with the SEPA payment method. However, we do need to enable _some_ additional LPMs on all split-UPE tests, because doing so automatically enables the multi-currency feature and adds EUR as a presentment currency, which later tests are dependent on. 

Secondly, there is a click action on a fraud protection tools tour button that is triggered inconsistently. I made this action a little more precise by using the specific class mentioned in WooCommerce's own E2E tests and adding a more robust function to ensure the button is always located and definitively clicked. 




#### Testing instructions


1. You are welcome to pull these tests locally, configure your own local E2E testing setup, and run these tests; but a more effective and accurate method would be to keep refreshing the testing jobs on this PR and ensure that they always come back fully green.
2. In either case, the tests should now always pass.



-------------------


**Post merge**
